spec/unit/pdk/module/update_manager_spec.rb

```diff
@@ -98,14 +98,14 @@
           end
 
           it 'removes the file' do
-            expect(FileUtils).to receive(:rm).with(dummy_file)
```

Comment:
Hrmm,  this commit is a bit odd.  Only tests were modified, nothing actually in `/lib`

Reply:
Yeah, some of the methods had already been updated to use `PDK::Util::Filesystem` but the tests weren't updated at the time.